packages/changesets/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import { changelogFunctions } from "./changelog";
+
+export default changelogFunctions;
```

Comment:
Nitpick: I think you could also write it as 
```suggestion
export { changelogFunctionsa as default } from "./changelog";
```

Reply:
Initially I commited the suggestion, but then I saw that it conflicts with `no-restricted-exports` rule
https://eslint.org/docs/latest/rules/no-restricted-exports